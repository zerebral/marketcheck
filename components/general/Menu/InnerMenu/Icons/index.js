import Account from './Account'
import Search from './Search'

import styled from 'styled-components'

const Icons = styled.div`
  display: flex;
  align-items: center;
  margin-left: auto;
`

export default () => (
  <Icons>
    <Search />
    <Account />
  </Icons>
)
