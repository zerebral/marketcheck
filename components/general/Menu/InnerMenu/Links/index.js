import Logo from './Logo'
import Wrapper from './Wrapper'
import Item from './Item'
import Link from './Link'
import links from './links'
import styled from 'styled-components'

const SpanBeta = styled.span`
  color: #fd4b2f;
`
const LinkDiv = styled.div`
  color: #fd4b2f;
`

export default () =>
  <Wrapper style={{width: "100%"}}>
    <Item  style={{width: "30%"}} logo>
        <Link href='/'><Logo /><SpanBeta>Beta</SpanBeta></Link>
    </Item>
      <LinkDiv  style={{width: "70%"}}>
    {links.map((link, index) => (
      <Item  style={{float: "right"}} key={link.name + index}>
        <Link href={link.route} target={link.target}>{link.name}</Link>
      </Item>
    ))}
      </LinkDiv>
  </Wrapper>
