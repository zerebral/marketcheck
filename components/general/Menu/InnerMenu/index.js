import React from 'react'
import Links from './Links'
import Icons from './Icons'
import { Container } from '~/layout'

import styled from 'styled-components'

import { em } from 'polished'

const InnerMenu = styled.div`
  display: flex;
  padding: ${em(20)} 0;
`

export default () => (
  <Container>
    <InnerMenu>
      <Links />
      {/*<Icons />*/}
    </InnerMenu>
  </Container>
)
