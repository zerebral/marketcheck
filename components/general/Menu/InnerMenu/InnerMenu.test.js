/* eslint-env jest */
import React from 'react'
import renderer from 'react-test-renderer'

import InnerMenu from './index'

describe('InnerMenu Component', () => {
  test('InnerMenu Passes the snapshot', () => {
    const component = renderer.create(<InnerMenu />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
