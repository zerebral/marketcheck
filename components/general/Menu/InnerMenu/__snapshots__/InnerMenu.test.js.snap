// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InnerMenu Component InnerMenu Passes the snapshot 1`] = `
<div
  className="Container-s1gwg6so-0 jOdNxa"
>
  <div
    className="InnerMenu-jwk4ke-0 hBYkzP"
  >
    <div
      className="Wrapper-s1tee10n-0 jTeLrU"
    >
      <div
        className="Item-j9flgi-0 cjeiJy"
      >
        <a
          className="Link-s74w1na-0 hOtkGw"
          href="/"
        >
          <svg
            className="Logo-s1hjwja6-0 hfOzNY"
            viewBox="0 0 142.85 24"
            xmlns="http://www.w3.org/2000/svg"
          >
            <title>
              logo
            </title>
            <g
              fill="#fff"
            >
              <path
                d="M2.44 5.11L7.13 16.8l4.68-11.69h2.45v14.33h-1.92v-5.58l.18-6-4.7 11.6H6.4L1.71 7.87l.19 6v5.58H0V5.11zM23.89 19.44a3.8 3.8 0 0 1-.26-1.12 4.06 4.06 0 0 1-3 1.32 3.76 3.76 0 0 1-2.58-.89 2.88 2.88 0 0 1-1-2.26 3 3 0 0 1 1.26-2.58 6 6 0 0 1 3.52-.91h1.77v-.84a2 2 0 0 0-.57-1.52 2.29 2.29 0 0 0-1.69-.58 2.65 2.65 0 0 0-1.63.49 1.45 1.45 0 0 0-.66 1.19h-1.82a2.51 2.51 0 0 1 .57-1.54A3.87 3.87 0 0 1 19.34 9a5.16 5.16 0 0 1 2.13-.43 4.21 4.21 0 0 1 2.87.95 3.32 3.32 0 0 1 1.08 2.53V17a6 6 0 0 0 .37 2.33v.16zm-3-1.39a3.2 3.2 0 0 0 1.62-.44 2.68 2.68 0 0 0 1.11-1.15v-2.19h-1.44q-3.35 0-3.35 2a1.66 1.66 0 0 0 .57 1.34 2.19 2.19 0 0 0 1.47.44zM33.42 10.43a5.46 5.46 0 0 0-.9-.07 2.4 2.4 0 0 0-2.43 1.53v7.56h-1.81V8.79h1.77v1.23a2.84 2.84 0 0 1 2.57-1.42 1.86 1.86 0 0 1 .81.14zM38.07 14.51l-1.14 1.19v3.74h-1.82V4.33h1.82v9.14l1-1.17 3.32-3.5h2.21l-4.12 4.44 4.63 6.2h-2.19zM49.73 19.64a4.65 4.65 0 0 1-3.52-1.42 5.28 5.28 0 0 1-1.36-3.8v-.33a6.39 6.39 0 0 1 .61-2.83 4.72 4.72 0 0 1 1.69-1.96 4.26 4.26 0 0 1 2.36-.7 4 4 0 0 1 3.22 1.4 5.92 5.92 0 0 1 1.15 3.92v.76h-7.21a3.78 3.78 0 0 0 .92 2.54 2.89 2.89 0 0 0 2.24 1 3.17 3.17 0 0 0 1.63-.39 4.12 4.12 0 0 0 1.17-1l1.11.87a4.47 4.47 0 0 1-4.01 1.94zm-.23-9.55a2.43 2.43 0 0 0-1.85.8 3.89 3.89 0 0 0-.93 2.25h5.33V13a3.47 3.47 0 0 0-.71-2.15 2.29 2.29 0 0 0-1.83-.76zM58.46 6.21v2.58h2v1.41h-2v6.61a1.48 1.48 0 0 0 .27 1 1.12 1.12 0 0 0 .91.32 4.46 4.46 0 0 0 .87-.12v1.47a5.33 5.33 0 0 1-1.4.2 2.27 2.27 0 0 1-1.84-.74 3.17 3.17 0 0 1-.62-2.1V10.2H54.7V8.79h1.94V6.21z"
              />
            </g>
            <rect
              fill="#13a54e"
              height="24"
              rx="5.48"
              ry="5.48"
              width="78"
              x="64.85"
            />
            <g
              fill="#fff"
            >
              <path
                d="M82.34 15a4.42 4.42 0 0 1-1.41 3.5 5.54 5.54 0 0 1-3.91 1.33 5.26 5.26 0 0 1-4.06-1.69 6.12 6.12 0 0 1-1.55-4.32v-2.7a6.19 6.19 0 0 1 1.54-4.32 5.15 5.15 0 0 1 4-1.7 5.78 5.78 0 0 1 4 1.29 4.38 4.38 0 0 1 1.39 3.54V10h-2.33a3 3 0 0 0-.78-2.2 3.13 3.13 0 0 0-2.28-.8 2.72 2.72 0 0 0-2.31 1.16 4.79 4.79 0 0 0-.85 2.92v2.75a4.77 4.77 0 0 0 .87 2.94 2.79 2.79 0 0 0 2.36 1.16 3 3 0 0 0 2.23-.77 3 3 0 0 0 .76-2.16zM86.76 10.37a3.79 3.79 0 0 1 1.29-1.16 3.47 3.47 0 0 1 1.69-.41 3.35 3.35 0 0 1 2.6 1.06 4.85 4.85 0 0 1 1 3.3v6.49h-2.4v-6.5a2.76 2.76 0 0 0-.49-1.82 1.81 1.81 0 0 0-1.45-.57 3 3 0 0 0-1.33.27 2.43 2.43 0 0 0-.92.75v7.86h-2.41V4.29h2.39zM100.34 19.85a4.64 4.64 0 0 1-3.65-1.47 5.51 5.51 0 0 1-1.35-3.82v-.39a5.7 5.7 0 0 1 1.31-3.86 4.25 4.25 0 0 1 3.39-1.51 4 4 0 0 1 3.3 1.32 5.35 5.35 0 0 1 1.1 3.54v1.29h-6.6a3.49 3.49 0 0 0 .69 2.13 2.35 2.35 0 0 0 2 .82 5.39 5.39 0 0 0 1.69-.24 4.82 4.82 0 0 0 1.32-.68l.78 1.57a5.09 5.09 0 0 1-1.61.9 6.78 6.78 0 0 1-2.37.4zm-.26-9.13a1.85 1.85 0 0 0-1.5.67 3.34 3.34 0 0 0-.7 1.76h4.16V13a2.69 2.69 0 0 0-.48-1.66 1.77 1.77 0 0 0-1.49-.63zM110.7 17.94a2 2 0 0 0 1.38-.51 1.68 1.68 0 0 0 .57-1.3h2.17v.06a3.21 3.21 0 0 1-1.19 2.58 4.75 4.75 0 0 1-6.5-.44 5.84 5.84 0 0 1-1.25-3.83v-.34a5.85 5.85 0 0 1 1.25-3.83 4.32 4.32 0 0 1 3.54-1.53 4.18 4.18 0 0 1 3 1.12 3.65 3.65 0 0 1 1.12 2.84v.06h-2.17a2.16 2.16 0 0 0-.54-1.49 1.81 1.81 0 0 0-1.42-.61 2 2 0 0 0-1.84 1 4.9 4.9 0 0 0-.56 2.45v.34a4.94 4.94 0 0 0 .58 2.49 2 2 0 0 0 1.86.94zM119.23 15.28v4.36h-2.38V4.29h2.38v7l2.7-4h2.8z"
              />
            </g>
            <path
              d="M122.24 13.65l4.34 4.94-1.41-.04 16.68-14.27-15.31 15.73-.68.69-.74-.74-4.27-3.68 1.39-2.63z"
              fill="#fff"
            />
          </svg>
        </a>
      </div>
      <div
        className="Item-j9flgi-0 cnLbmF"
      >
        <a
          className="Link-s74w1na-0 hOtkGw"
          href="#"
        >
          Home
        </a>
      </div>
      <div
        className="Item-j9flgi-0 cnLbmF"
      >
        <a
          className="Link-s74w1na-0 hOtkGw"
          href="#"
        >
          Research
        </a>
      </div>
      <div
        className="Item-j9flgi-0 cnLbmF"
      >
        <a
          className="Link-s74w1na-0 hOtkGw"
          href="#"
        >
          About
        </a>
      </div>
      <div
        className="Item-j9flgi-0 cnLbmF"
      >
        <a
          className="Link-s74w1na-0 hOtkGw"
          href="#"
        >
          FAQ
        </a>
      </div>
      <div
        className="Item-j9flgi-0 cnLbmF"
      >
        <a
          className="Link-s74w1na-0 hOtkGw"
          href="#"
        >
          For Dealers
        </a>
      </div>
    </div>
    <div
      className="Icons-s15rosyg-0 kmRZS"
    >
      <a
        className="Search__A-r2sj56-0 injnFb"
      >
        <svg
          className="Search-r2sj56-1 ixjDcK"
          viewBox="0 0 14.86 14.62"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M14.15 14.62l-4.4-4.48a5.83 5.83 0 1 1 .68-.73l4.43 4.51zM5.83 1a4.83 4.83 0 1 0 4.83 4.83A4.84 4.84 0 0 0 5.83 1z"
          />
        </svg>
      </a>
      <a
        className="Account__A-atchjn-0 bSTpxW"
      >
        <svg
          className="Account__User-atchjn-1 kGfKyf"
          viewBox="0 0 12.36 15.64"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M6.18 7.86a3.93 3.93 0 1 0-3.93-3.93 3.93 3.93 0 0 0 3.93 3.93zm0-6.86a2.93 2.93 0 1 1-2.93 2.93A2.93 2.93 0 0 1 6.18 1zm6.18 14.64h-1a5.18 5.18 0 0 0-10.36 0H0a6.18 6.18 0 0 1 12.36 0z"
            fill="#fff"
          />
        </svg>
      </a>
    </div>
  </div>
</div>
`;
